packages/features/ee/sso/lib/sso.ts

```diff
@@ -0,0 +1,57 @@
+import type { PrismaClient } from "@calcom/prisma";
+import { TRPCError } from "@calcom/trpc/server";
+
+import jackson from "./jackson";
+import { tenantPrefix, samlProductID } from "./saml";
+
+export const ssoTenantProduct = async (prisma: PrismaClient, email: string) => {
+  const { connectionController } = await jackson();
+
```

Comment:
- Find the teams the user belongs to from the membership table
- Find the sso connection for the all the teams the user belongs to
- Find the first sso connection that is enabled for the user